cmis_web_report_write_alf/static/src/js/cmis_folder.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+/* ---------------------------------------------------------
++ * Odoo cmis_web_bus
++ * Authors Laurent Mignon 2016, Maxime Franco 2023 Acsone SA/NV
++ * License in __openerp__.py at root level of the module
++ *---------------------------------------------------------
++*/
+
+import {CmisFolderField} from "@cmis_web/cmis_folder/cmis_folder";
+import {patch} from "web.utils";
+
+patch(CmisFolderField.prototype, "cmis_folder_open_url", {
+    _handleNotify(event) {
+        this._super(event);
+        if (this.rootFolderId !== event.detail.cmis_objectid) {
```

Comment:
Where does `event.detail.cmis_objectid` comes from ?

Reply:
Inside cmis_web_bus, in the python code when an event is triggered, there are some informations to identify the correct CmisFolder:
`        bus_message = {
            "cmis_objectid": cmis_objectid,
            "backend_id": cmis_backend.id,
            "action": action,
            "user_id": self.env.user.id,
        }`